recipes/recipe-Ayurvedic-Kitchari.html

```diff
@@ -0,0 +1,171 @@
+        <link rel="icon" type="image/png" sizes="32x32" href="../images/favicon/favicon-32x32.png">
+        <link rel="icon" type="image/png" sizes="16x16" href="../images/favicon/favicon-16x16.png">
+        <link rel="manifest" href="../images/favicon/site.webmanifest">
+        <link rel="mask-icon" href="../images/favicon/safari-pinned-tab.svg" color="#194a40">
+        <meta name="msapplication-TileColor" content="#194a40">
+        <meta name="theme-color" content="#194a40">
+    </head>
+
+    
+    <body> <main>
+        <div class="container">
+
+            <nav>
+                <a id="logo" href="../index.html"><img src="../images/logo-black.png" alt="Logo"/></a>
+                <ul>
+                    <li><a href="../index.html">Home</a></li>
+                    <li><a href="../recipes.html" class = "currentpage">Recipes</a></li>
+                    <li><a href="../challenge.html">Challenge</a></li>
+                    <li><a href="../resources.html">Resources</a></li>
+                    <li><a href="../contact.html">Contact</a></li>
+                </ul>
+            </nav>
+
+            
+            <div class="featured_recipe" id="recipe_page">
+                <div class="title_text">
+                    <a href="../recipes.html#recipes"><i class="fas fa-chevron-left"></i></a>
+                    <h2>Ayurvedic Kitchari</h2>
+                </div>
+            </div>
+                
+            <div class="recipe_image_text_section">
+                <img class="photo left" src="../images/recipes/Ayurvedic-Kitchari.jpeg" alt="Recipe"/>
+<!--                <iframe class="recipe-video left" src="https://www.youtube.com/embed/hSJuW2qgmfk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>-->
+                <div class="recipe_body right">
+                    <div>
+                        <h3>Recipe submitted by Stanley</h3>
+                        <p class="recipe_info">
+                            <i class="fas fa-seedling"></i>Vegan&ensp;   
+                            <i class="fas fa-carrot"></i>Vegetarian&ensp;
+                            <i class="fas fa-ban"></i>Gluten-Free&ensp;
+                            <i class="fas fa-circle"></i>Easy&ensp;
+    <!--                        <i class="fas fa-square"></i>Moderate&ensp;-->
+    <!--                        <i class="fas fa-star"></i>Difficult&ensp;-->
+                            <br>
+                            <i class="fas fa-clock"></i>55 mins&ensp;
+                            <i class="fas fa-user"></i>3 Servings
+                        </p>
+<!--
+                        <p>Description: 
+                            <a href="https://www.youtube.com/watch?v=tUi4_ektVn0" target="_blank"> Video 1</a>
+                        </p>
+-->
```

Comment:
```suggestion

                        <p>Description: This healing Kitchari is so nourishing and easy on the stomach. It is the perfect "feel better", easy to digest meal when your body needs a break! Kitchari can be used for a simple 3-, 10- or 20 day cleanse.
                            <a href="https://www.youtube.com/watch?v=tUi4_ektVn0" target="_blank"> Video 1</a>
                        </p>

```

Reply:
Missing the "Description" and "Additional Information" sections in Excel, always want to make sure all the information is included. These can usually go in the description area.